docs/src/resample_proj.md

```diff
@@ -0,0 +1,124 @@
+## resample and Projections with ProjString
+
+Geospatial datasets will come in different [projections](https://proj.org/en/9.4/operations/projections/index.html) for whaever reasons. Here, we will focus in some of the most used ones, e.g., `MODIS SINUSOIDAL` and `EPSG`, as well as transformations between them.
+
+Let's start by loading the neccesary packages:
+
+````@example modis
+using Rasters, RasterDataSources, ArchGDAL
+using DimensionalData
+using DimensionalData.Lookups
+using NaNStatistics
+using CairoMakie
+````
+
+and let's load a test raster
+
+````@example modis
+ras = Raster(WorldClim{BioClim}, 5)
+ras_m = replace_missing(ras, missingval=NaN)
+````
+
+and let's also take a look
+
+````@example modis
+fig = plot(ras_m; colorrange=(0,100))
+````
+
+### MODIS SINUSOIDAL PROJECTION
+
+````@example
+# ? is this the right ProjString ?, do we need to shift lat, lon?
+SINUSOIDAL_CRS = ProjString("+proj=sinu +lon_0=0 +x_0=0 +y_0=0 +a=6371007.181 +b=6371007.181 +units=m +no_defs")
+````
+
+and hence the `resample` is performed with
+
+````@example modis
+ras_sin = resample(ras_m; size=(1440,720), crs=SINUSOIDAL_CRS, method="sum")
+````
+
+let's compare the total counts!
+
+````@example modis
+nansum(ras_m), nansum(ras_sin)
+````
+
+and, how does this looks like?
+
+````@example modis
+fig = plot(ras_sin; colorrange=(0,100))
+````
+
+now, let's go back to `latitude` and `longitude`
+
+````@example modis
+# ? also here, do we need to shift X, Y?
+ras_epsg = resample(ras_sin; size=(1440,720), crs=EPSG(4326), method="sum")
+````
+
+and let's apply `shiftlocus` such that we can harmonize coordinates, which might be needed when building bigger datasets:
```

Comment:
what does this do, what does it mean to harmonize coordinates?

Reply:
I usually need to match to other variables, which are in a specific Lon/lat range.